package.json

```diff
@@ -400,6 +400,8 @@
         "webpack-cli": "^3.3.7"
     },
     "dependencies": {
+        "@octokit/rest": "^18.0.1",
+        "@octokit/types": "^5.1.0",
```

Comment:
The point of making it a dev dependency is so that we don't ship it with our vsix and put it on every user's machine. That's more important than some tslint rule

The quick/dirty method would be to disable the rule line-by-line, but the best way to do it is to add "@octokit/types" to the list of allowed implicit dependencies here: https://github.com/microsoft/vscode-azurestaticwebapps/blob/master/tslint.json#L59